core/src/test/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceServiceSpec.scala

```diff
@@ -95,7 +98,7 @@ class WorkspaceServiceSpec extends AnyFlatSpec with ScalatestRouteTest with Matc
     val gcsDAO = Mockito.spy(new MockGoogleServicesDAO("test", googleAccessContextManagerDAO))
     val samDAO = Mockito.spy(new MockSamDAO(dataSource))
     val gpsDAO = new MockGooglePubSubDAO
-    val workspaceManagerDAO = mock[MockWorkspaceManagerDAO](RETURNS_SMART_NULLS)
+    val workspaceManagerDAO = Mockito.spy(new MockWorkspaceManagerDAO())
```

Comment:
Using a mock here results in a mock with mocks on all methods methods on this class, not an instance of the `MockWorkspaceManagerDAO` supplying the already implemented mock methods. The existing usages of this mock look like they were after verifying calls to the external service, which lends itself to a spy vs a mock.